.releaserc

```diff
@@ -14,8 +11,8 @@
           {"type": "fix", "section": "Bug Fixes"},
           {"type": "perf", "section": "Performance Improvements"},
           {"type": "revert", "section": "Reverts"},
-          {"type": "chore", "section": "Miscellaneous Chores"},
-          {"type": "refactor", "section": "Code Refactoring"},
+          {"type": "chore", "section": "Miscellaneous Chores", "hidden": true},
+          {"type": "refactor", "section": "Code Refactoring", "hidden": true},
```

Comment:
Can hide only chore? (personally only hiding renovate would be preferred though)

So far, past refactoring tasks occasionally caused regressions/performance degrade (element lookup) we noticed by user reports. Keeping possible production code change in the changelog may help to guess what change might affect some behavior

Reply:
I suppose, but that information can be gleaned from other sources.  I'll revert it since I don't care that much